code/style.css

```diff
@@ -1,233 +1,362 @@
-  display: flex;
-  flex-direction: column;
-  padding: 5vw;
-  align-items: flex-start;
-  box-sizing: border-box;
+ .filled-button:hover {
+     background-color: white;
+     border-color: white;
+     color: var(--primary);
+     transition: .7s;
 }
 
-select {
-  display: block;
-  border: none;
-  font-size: 22px;
-  font-family: 'Montserrat';
-  color: var(--secondary);
-  width: 100%;
-  margin: 24px 0;
+ .outlined-button:hover {
+     background-color: var(--primary);
+     color: white;
+     transition: .7s;
 }
 
-.guess-who-icon {
-  width: 126px;
+ .question-count p {
+     color: var(--primary);
+     font-size: 2.5rem;
 }
 
-.board-wrapper {
-  width: 70%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
+ .timer-game p {
+     color: var(--white);
+     font-size: 2.5rem;
 }
 
-.game-board {
-  width: 90%;
-  max-width: 882px;
-  display: flex;
-  flex-wrap: wrap;
-  align-content: center;
+ #question-slide > * {
+     margin-bottom: 3rem;
+}
+
+ #question-slide > :last-child {
+     margin-bottom: 0rem;
+}
+
+/* ===== BOARD ===== */
+
+ .board-wrapper {
+     width: 70%;
+     display: flex;
+     align-items: center;
+     justify-content: center;
+     background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+     animation: gradient 15s ease infinite;
+     background-size: 500% 500%;
+     padding: 4rem 2.5rem;
+}
+
+ @keyframes gradient {
+     0% {
+         background-position: 0% 50%;
+    }
+     50% {
+         background-position: 100% 50%;
+    }
+     100% {
+         background-position: 0% 50%;
+    }
+}
+
+ .game-board {
+     width: 100%;
+     max-width: 89rem;
+     flex-wrap: wrap;
+     align-content: space-between;
+     justify-content: space-evenly;
 }
 
 /****** CARD ******/
-.card {
-  width: 135px;
-  height: 165px;
-  overflow: hidden;
-  border: 3px solid var(--secondary);
-  border-radius: 4px;
-  margin: 3px;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: space-between;
+
+ .card {
+     width: 12rem;
+     height: 16rem;
+     overflow: hidden;
+     background-color: var(--white);
+     border-radius: 1rem;
+     margin: 1rem;
 }
 
-.card p {
-  text-align: center;
-  margin-bottom: 0;
-  font-size: 16px;
-  font-weight: 700;
+ .card p {
+     text-align: center;
+     font-size: 2rem;
+     letter-spacing: .1rem;
+     font-weight: 300;
 }
 
-.card .guess {
-  display: none;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  color: white;
-  text-align: center;
-  height: inherit;
+ .card .guess {
+     display: none;
+     flex-direction: column;
+     align-items: center;
+     justify-content: center;
+     color: white;
+     text-align: center;
+     height: inherit;
+     letter-spacing: .1rem;
+     font-weight: 300;
 }
 
-.card:hover {
-  background-color: var(--secondary);
+ .card:hover {
+     background-color: var(--secondary);
+     transition: 1s;
```

Comment:
Mobile mode as well as DevTools responsive mode: I don´t know if it´s something in the hover function or what it is that makes the whole card turn black if you click it (since you are not able to hover on mobile or in responsive mode in DevTools). There is no white text shown "Guess on Nadia" either (as it is on PC-version). I´m sure we will learn how to handle hover functions for mobiles and ipads later on in the course :)